Vagrantfile

```diff
@@ -0,0 +1,71 @@
+# -*- mode: ruby -*-
```

Comment:
I'm not sure how this got into the repo, but it shouldn't be necessary since we use Homestead for this project.